experimental/dds/attributable-map/src/interfaces.ts

```diff
@@ -19,7 +19,6 @@ export interface IValueChanged {
 	 * The value that was stored at the key prior to the change.
 	 */
 	// TODO: Use `unknown` instead (breaking change).
-	// eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
How come these went away? Was the rule already disabled at the package level?

Reply:
Good catch; I don't know why these changes were made or why they are considered OK. Seems like a bug in either eslint or the rule itself. I have
reverted these changes.